expression/integration_test.go

```diff
@@ -4909,3 +4909,12 @@ func (s *testIntegrationSuite) TestIssue11309And11319(c *C) {
 	tk.MustQuery(`SELECT DATE_ADD('2007-03-28 22:08:28',INTERVAL 2.2 DAY_HOUR)`).Check(testkit.Rows("2007-03-31 00:08:28"))
 	tk.MustQuery(`SELECT DATE_ADD('2007-03-28 22:08:28',INTERVAL 2.2 YEAR_MONTH)`).Check(testkit.Rows("2009-05-28 22:08:28"))
 }
+
+func (s *testIntegrationSuite) TestIssue11943(c *C) {
```

Comment:
You can put it in the `TestWindowFunction` of plan package or `TestWindowFunctions` of executor package.

Reply:
ok